minithesis.py

```diff
@@ -566,7 +640,8 @@ def shrink(self):
         # not to work.
         cached = CachedTestFunction(self.test_function)
 
-        def consider(choices):
+        def consider(choices: Sequence[int]) -> bool:
+            # XXX This assumes choices and self.result have the same type
```

Comment:
I think they are always the same type in practice. Is there a reason not to use the more precise `array[int]` here for `choices`?

Reply:
Oh, that's right. I think the rule is that *prefix* may be a list, but *choices* and *result* are always `array`. So I'll type it like that, it appears to work.